.gitignore

```diff
@@ -1 +1,2 @@
 .DS_Store
+serverless-application-model.code-workspace
```

Comment:
What is this for?

Reply:
Accidentally checked this in. It's removed now.